### DISCUSSION TITLE
New Log in WinLogBeat Not Showing Up

### DISCUSSION THREAD

**Discussion by lonewaffle** [Security-Onion-Solutions/securityonion#8163] [Unsupported Versions]
I've added a new log to our WinLogBeat yaml file to ship to Security Onion (Microsoft-Windows-NTLM/Operational, so I can audit this in preparation for blocking NTLM) and I can see that entries are being added to the log in even viewer:
![Screen_Shot_2022-06-16_at_12_43_20_PM](https://user-images.githubusercontent.com/5062130/174826499-b5918202-e203-4420-b3b1-77d207f39296.png)

And the logs are shipping according to the WinLogBeat log:
![Screen Shot 2022-06-16 at 12 42 52 PM](https://user-images.githubusercontent.com/5062130/174826540-e4be4d2c-207f-4ed2-90b7-a40e3e0b23d0.png)

But I'm not seeing any of them show up in Security Onion. Am I missing something?
![Screen_Shot_2022-06-16_at_12_25_33_PM](https://user-images.githubusercontent.com/5062130/174826796-37838246-b0a4-4854-9f46-c09b1c935c52.png)

I'm using Security Onion 2.3.110 and WinLogBeat 7.11.2. Can someone point out what I forgot to do?

Thanks!
-Josh

**Comment by lonewaffle** [comment]
I'm able to find these logs when I search in Kibana. I'm guessing the Hunt section only includes specific logs.

**Reply by lonewaffle** [reply]
Nevermind, I'm not seeing entries from the computer in question. So it seems that some computers in my environment are sending these and they make it in, but some are not, even though the beat indicates that they are.

**Comment by weslambert** [comment]
You might try checking for errors in the Logstash log in `/opt/so/log/logstash` on your search node(s).

**Reply by lonewaffle** [reply]
Looking at the logstash log, it appears my problem would be this:

response=>{"index"=>{"_index"=>"so-beats-2022.06.27", "_type"=>"_doc", "_id"=>"YzenpoEBFc4F3-ja37X7", "status"=>400, "error"=>{"type"=>"mapper_parsing_exception", "reason"=>"failed to parse", "caused_by"=>{"type"=>"illegal_argument_exception", "reason"=>"Limit of total fields [1500] has been exceeded while adding new fields [2]"}}}}}

I ran the following commands last Thursday to try to correct this:

sudo so-elasticsearch-query template/so-beats -d '{"index.mapping.total_fields.limit": 2000}'

sudo so-elasticsearch-query so-beats-2022.06.23/_settings -d '{"index.mapping.total_fields.limit": 2000}' -XPUT

It looks like it's still limited to 1500 fields, so I guess my attempt to expand it didn't work.

**Comment by weslambert** [comment]
What version of Security Onion are you running? In the latest version, the default field limit  for  the `so-beats` index is `5000`, and you modify it in the pillar, like so:

```
index_settings:
  so_beats:
    index_template:
      template:
        settings:
          index:
            mapping:
              total_fields:
                limit: $MYLFIELDLIMIT
```

**Reply by lonewaffle** [reply]
Sorry for the delay. I updated to 2.3.130 right after I posed that last message about the 1500 fields being exceeded. Noticed that it upped to 5000, but now I'm seeing the same message saying that 5000 fields have been exceeded. Seems hard to believe I'm using that many fields. 

Thanks for telling me the right way to adjust this setting! I'll play with it a bit and see if I can make it happy.
